### PR TITLE
add text on aal

### LINKED ISSUE #15: Security levels in the attestation
Since the EUDI Wallet instance may have different level of security depending on its own hardware and/or software capabilities, here the proposal: define a claim name for exposing different level of security of the wallet instance.

Example:"gold", "silver" and "bronze" or "high", "medium", "low" depending on the framework used. 

This proposal enables the security evaluation to the credential issuing side (or any other Relying Party) without knowing any specific hardware or software features (e.g. OS, etc.).

### LINKED ISSUE #29: [IANA Registry] attested_security_context as JWT parameter
The brand new parameter called `attested_security_context`, as defined [here](https://github.com/vcstuff/draft-looker-oauth-attestation-based-client-auth/pull/26/files#diff-55e05bec82593426ed363ab5c93ddf1e82b0f64cdf164f98e85fe1236b8783edR325), requires a IANA section for the registration of this parameter name

### PR DESCRIPTION
**Pull request by paulbastian**
Closes #15 
Closes #29 

## 📑 Description
Add text on aal claim

## Preview Link

[click here for rendered preview of PR](https://github.com/<repo-name>/<branch-name>/<draft-name>.html)

### CODE REVIEW

File: draft-ietf-oauth-attestation-based-client-auth.md [hunk 1]
```diff
@@ -188,11 +188,13 @@ The following rules apply to validating the client attestation JWT. Application
 
 7. The JWT MAY contain a "jti" (JWT ID) claim that provides a unique identifier for the token.
 
-8. The JWT MAY contain other claims.
+8. The JWT MAY contain a "aal" (authenticator assurance level) claim that describes the attested assurance level of the Client Instance and the Client Instance Key. The claim value contains a URL that references a jurisdictional framework. Consumers of the client attestation are RECOMMENDED to evaluate this claim, if they intend to state an assurance level for their issued tokens themselves.
```
Comment (paulbastian) [inline draft-ietf-oauth-attestation-based-client-auth.md:191]:
```suggestion
8. The JWT MAY contain a "aal" (authenticator assurance level) claim that describes the attested assurance level of the Client Instance and the Client Instance Key. The claim value contains a URI that references a jurisdictional framework. Consumers of the client attestation are RECOMMENDED to evaluate this claim, if they intend to state an assurance level for their issued tokens themselves.
```

**Comment by peppelinux** [comment]
Supported here
https://github.com/italia/eudi-wallet-it-docs/pull/155

anyway it would be better define within the specs the grouping of the security capabilities according to the levels low, medium and high

File: draft-ietf-oauth-attestation-based-client-auth.md [hunk 2]
```diff
@@ -188,11 +188,13 @@ The following rules apply to validating the client attestation JWT. Application
 
 7. The JWT MAY contain a "jti" (JWT ID) claim that provides a unique identifier for the token.
 
-8. The JWT MAY contain other claims.
+8. The JWT MAY contain a "aal" (authenticator assurance level) claim that describes the attested assurance level of the Client Instance and the Client Instance Key. The claim value contains a URI that references a jurisdictional framework. Consumers of the client attestation are RECOMMENDED to evaluate this claim, if they intend to state an assurance level for their issued tokens themselves.
```
Comment (peppelinux) [inline draft-ietf-oauth-attestation-based-client-auth.md:191]:
```suggestion
8. The JWT MAY contain a "aal" (authenticator assurance level) claim that describes the attested assurance level of the Client Instance and the Client Instance Key. The claim value contains a URI that references a jurisdictional framework. Consumers of the Client attestation are RECOMMENDED to evaluate this claim, if they intend to state an assurance level for their issued tokens themselves.
```

**Review by peppelinux — APPROVED** [review]

**Comment by paulbastian** [comment]
regarding the current ideas developed in OpenID4VCI to separate wallet attestation and key attestation (see https://github.com/openid/OpenID4VCI/issues/355 ) this no longer makes sense here

**Pull request closed without merging**